pafy/backend_shared.py

```diff
@@ -4,6 +4,7 @@
 import time
 import logging
 import subprocess
+import random
```

Comment:
I don't think we need to import `random`, or do we need to use it somewhere?

Reply:
I was planing to port the fix over. This is an artifiact of that!